src/biogeophys/LunaMod.F90

```diff
@@ -402,18 +404,22 @@ subroutine Update_Photosynthesis_Capacity(bounds, fn, filterp, &
                          PNcbold   = 0.0_r8                                     
                          call NitrogenAllocation(FNCa,forc_pbot10(p), relh10, CO2a10, O2a10, PARi10, PARimx10, rb10v, hourpd, &
                               tair10, tleafd10, tleafn10, &
-                              Jmaxb0, Jmaxb1, Wc2Wjb0, relhExp, PNlcold, PNetold, PNrespold, &
+                              Jmaxb0, Jmaxb1, Wc2Wjb0, relhExp, PNlcold, PNetold, PNrespold, dayl_factor(p),&
```

Comment:
@lmbirch89 @olyson noticed the order these arguments is wrong. Based on the subroutine, daylfactor(p), should be after the next argument PNcbold. That could have messed up your results if it was in place like this. So you should look into that. And you might need to redo some of your experiments.

Here's the fix that I'm putting into the branch that we are testing this on.

``` fortran
diff --git a/src/biogeophys/LunaMod.F90 b/src/biogeophys/LunaMod.F90
index 2ca8406a..9087d1e1 100644
--- a/src/biogeophys/LunaMod.F90
+++ b/src/biogeophys/LunaMod.F90
@@ -402,8 +402,8 @@ subroutine Update_Photosynthesis_Capacity(bounds, fn, filterp, &
                          PNcbold   = 0.0_r8                                     
                          call NitrogenAllocation(FNCa,forc_pbot10(p), relh10, CO2a10, O2a10, PARi10, PARimx10, rb10v, hourpd, &
                               tair10, tleafd10, tleafn10, &
-                              Jmaxb0, Jmaxb1, Wc2Wjb0, relhExp, PNlcold, PNetold, PNrespold, dayl_factor(p), &
-                              PNcbold, PNstoreopt, PNlcopt, PNetopt, PNrespopt, PNcbopt)
+                              Jmaxb0, Jmaxb1, Wc2Wjb0, relhExp, PNlcold, PNetold, PNrespold, PNcbold, dayl_factor(p), &
+                              PNstoreopt, PNlcopt, PNetopt, PNrespopt, PNcbopt)
                          vcmx25_opt= PNcbopt * FNCa * Fc25
                          jmx25_opt= PNetopt * FNCa * Fj25
```

Reply:
Hi! Thanks for catching this. I had moved daylfactor around based on previous suggestions of sorting into inputs/outputs, but in my original model runs, daylfactor was the last argument.

Reply:
Ahh, OK, good to know. As long as the results you have been looking at had it in it correctly.